### PR TITLE
Create canadian-journal-of-earth-science.csl

### PR DESCRIPTION
**Pull request by libora**
One of the independent styles for NRC journals; this one has no linked dependent styles.

### CODE REVIEW

**Comment by adam3smith** [comment]
perfect, thanks!

File: canadian-journal-of-earth-science.csl [hunk 1]
```diff
@@ -0,0 +1,228 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" default-locale="en-US" version="1.0" demote-non-dropping-particle="sort-only">
+  <info>
+    <title>Canadian Journal of Earth Sciences</title>
+    <id>http://www.zotero.org/styles/canadian-journal-of-earth-science</id>
+    <link href="http://www.zotero.org/styles/canadian-journal-of-earth-science" rel="self"/>
+    <link href="http://www.zotero.org/styles/canadian-geotechnical-journal" rel="template"/>
+    <link href="http://www.nrcresearchpress.com/page/cgj/authors" rel="documentation"/>
```
Comment (rmzelle) [inline canadian-journal-of-earth-science.csl]: @libora, this still points to the page for the "Canadian Geotechnical Journal".

File: canadian-journal-of-earth-science.csl [hunk 2]
```diff
@@ -0,0 +1,228 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" default-locale="en-US" version="1.0" demote-non-dropping-particle="sort-only">
+  <info>
+    <title>Canadian Journal of Earth Sciences</title>
+    <id>http://www.zotero.org/styles/canadian-journal-of-earth-science</id>
```
Comment (rmzelle) [inline canadian-journal-of-earth-science.csl]: @libora, this also needs to be "sciences", not "science". (http://www.nrcresearchpress.com/journal/cjes)

Comment (rmzelle) [inline canadian-journal-of-earth-science.csl]: Fixed in https://github.com/citation-style-language/styles/pull/2083.

File: canadian-journal-of-earth-science.csl (hunk 1 above)
Comment (rmzelle) [inline canadian-journal-of-earth-science.csl]: Fixed in https://github.com/citation-style-language/styles/pull/2083.

**Comment by rmzelle** [comment]
Not perfect, but thanks all the same :).

File: canadian-journal-of-earth-science.csl (hunk 2 above)
Comment (libora) [inline canadian-journal-of-earth-science.csl]: Sorry for that mistakes.

Comment (rmzelle) [inline canadian-journal-of-earth-science.csl]: No problem.

**Pull request merged**